spec/spec.md

```diff
@@ -1176,6 +1175,28 @@ When using this [[ref:Feature]]:
   property. If present, its value ****MUST**** be a
   [JSON LD Framing Document](https://w3c.github.io/json-ld-framing/) object.
 
+### Retention Feature
+
+The Retention [[ref:Feature]] extends the [[ref:Input Descriptor Object]]'s _field_ 
+object, allowing a Verifier to indicate it will retain the submitted value for the
+specific field.    
+
+A [[ref:Verifier]] must not retain any data, including digests, signatures,
+and derived data received, except for claims from the [[ref:Presentation Submission]] descriptors
+for which the accompanying [[ref:Input Descriptor Object]]'s _field_ `intent_to_retain`
+property was set to true.
+
+Retain is defined as: “to store for a period longer than necessary to conduct the
+exchange between [[ref:Holder]] and [[ref:Verifier]] in realtime”.
+
+When using this [[ref:Feature]]:
+- `intent_to_retain` - The _fields_ object inside an [[ref:Input Descriptor Object]] ****MAY****
+ contain an `intent_to_retain` property.
+ If present, its value ****MUST**** be a boolean that indicates the [[ref:Verifier]]
+ intents to retain the [[ref:Claim]]'s data being requested. 
+ If not present, this feature is not enabled, which equates to `intent_to_retain` being `false` 
```

Comment:
does this force the feature, by implicitly making its missing value false

Reply:
Line has been removed completely as a Feature will be there or it won't